parser/parser.y

```diff
@@ -5532,7 +5532,7 @@ NumericType:
 		x := types.NewFieldType($1.(byte))
 		x.Flen = fopt.Flen
 		if x.Tp == mysql.TypeFloat {
-			if x.Flen > mysql.PrecisionForFloat {
+			if x.Flen > 24 {
```

Comment:
should we extract `24` as to a variable like `maxFloatScale` ?

Reply:
I think we'd better not define it as a constant to avoid being abused.
This is not a meaningful number got from MySQL manual,
but extracted from the behavior of MySQL. (#351 may be referred.)